### PR TITLE
Adding sample template for a kinesis stream

### PR DESCRIPTION
**Pull request by gneokleo**
- Adding an example for how to get metrics for a kinesis
    stream. This is in addition to getting metrics for a kinesis
    application.

### CODE REVIEW

File: sample_templates/kinesis.yml.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This is an example template file for getting metrics for a kinesis stream.
+# If you want an example of a kinesis application look at kinesisapp.yml.js.
```
Comment (crccheck) [inline sample_templates/kinesis.yml.js]: Is there supposed to `kinesisapp.yml.js` file here too?

Comment (gneokleo) [inline sample_templates/kinesis.yml.js]: I added that on a previous commit. It should be under sample_templates. I wanted to add this example too for people who want to get overall kinesis stream metrics (that are independent of kinesis application collectors)

Comment (crccheck) [inline sample_templates/kinesis.yml.js]: oh, it's already there! haha :derp:

Comment (gneokleo) [inline sample_templates/kinesis.yml.js]: thanks for the merge!

**Pull request merged**